cypress/e2e/landingPage.spec.cy.js

```diff
@@ -1,27 +1,128 @@
-/// <reference types="cypress" /> // is this really needed?
-import { onLandingPage } from "../support/pageObject/landingPage";
-import { onSignUpPage } from "../support/pageObject/signUpPage";
+import { faker } from "@faker-js/faker";
+// import { onLandingPage } from "../support/pageObject/landingPage";
+// import { onSignUpPage } from "../support/pageObject/signUpPage";
+
+const USERNAME = faker.person.firstName() + faker.number.int({ min: 1, max: 100 });
```

Comment:
you don't need to add number at the end, `faker` will generate all the time different values